packages/playground/src/components/user-management/form-user-delete-dialog.tsx

```diff
@@ -40,9 +28,9 @@ export const FormDeleteUserDialog: React.FC<FormResetPasswordrDialogProps> = ({
       <AlertDialogTrigger asChild></AlertDialogTrigger>
       <AlertDialogContent>
         <AlertDialogHeader>
-          <AlertDialogTitle>Are you absolutely sure to remove {user.display_name}?</AlertDialogTitle>
+          <AlertDialogTitle>Are you absolutely sure to remove {user?.display_name}?</AlertDialogTitle>
```

Comment:
`Are you sure you want to remove the user "{user?.display_name}"?`

Reply:
fixed